ui-components/src/locales/general.json

```diff
@@ -637,6 +637,7 @@
       "reEnterEmail": "Re-enter email address",
       "reEnterPassword": "Re-enter your password",
       "resendTheEmail": "Resend the email",
+      "emailSubscription": "Email me whenever a new listing is posted",
```

Comment:
Where is this wording from? I wonder if we want some more generic language since we'll also be adding emails for when listings are updated

Reply:
This was the wording that was in the mock in the issue, its pretty easy to change though if you think we should change it to be more generic

Reply:
Would this be better? "Email me whenever a new listing is posted or updated."

Reply:
Yeah that's better. let's do that for now and double check with Justin/Safiya